abletonosc/device.py

```diff
@@ -90,5 +128,7 @@ def device_get_parameter_name(device, params: Tuple[Any] = ()):
             return params[0], device.parameters[params[0]].name
 
         self.osc_server.add_handler("/live/device/get/parameter/value", create_device_callback(device_get_parameter_value))
+        self.osc_server.add_handler("/live/device/get/parameter/value_string", create_device_callback(device_get_parameter_value_string))
```

Comment:
Add some documentation about this function. I've not used Live's `str_for_value`; is this for human-readable representations of values, for UI etc?